contracts/MetadataViews.cdc

```diff
@@ -129,4 +129,38 @@ pub contract MetadataViews {
             return "ipfs://".concat(self.cid)
         }
     }
+
+    // A view to represent Media, a file with an correspoiding mediaType.
+    // media-type comes on the form of type/subtype as described here https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types
+    pub struct Media {
+        pub let file: AnyStruct{File}
+        pub let mediaType: String
+
+        init(file: AnyStruct{File}, mediaType: String) {
+          self.file=file
```

Comment:
Is there any sort of verification pre-condition we can do for `mediaType`?

Reply:
would this work?

```
 pre {             
  mediaType.utf8.contains("/".utf8[0]) : "Media type must be on the form type/subtype"
 }
```

Basically check the the media type contains a /.

Reply:
yeah, I think that works

Reply:
I agree it doesn't do much, but it at least will catch a few. But it does add three lines of code, which I guess might be more noise than it is worth